src/main/java/com/ecwid/consul/ConsulException.java

```diff
@@ -7,9 +7,6 @@
  */
 public class ConsulException extends RuntimeException {
 
-	public ConsulException() {
```

Comment:
This is a breaking change. I request that this be reverted.

Reply:
No-arg constructor is recovered but I think removing it is more normative.
if not declared parameter-construct java has a no-arg constructor built in it